### ISSUE TITLE
Deploying in Filewave

### ISSUE THREAD

**Issue by auslaner** [patgmac/Filewave-macOS-Updates#3]
I'm a little confused how the described setup in the README is meant to work in Filewave.

If the actual installation step takes place in the `fw_softwareupdate_activation.sh` script, and set as an activation script in the fileset, won't that only ever run a single time when the fileset is first associated?

As far as I know, verification scripts are the only ones that run on a regular basis but `fw_softwareupdates_verification.sh` only downloads the updates rather than installing them.

So just curios if I'm misunderstanding something here or if a fileset with these scripts is designed to be re-associated on a regular basis so that `fw_softwareupdate_activation.sh` will run again.

**Comment by patgmac** [comment]
You are absolutely right as I've eventually found. I'm fairly new to FW so the behavior of the various steps was new to me. Hoping to crowdsource a better solution if you have any ideas?

We have some professional services available to us and I've asked them to take a look as well.

**Comment by auslaner** [comment]
Gotcha, okay well that actually makes me feel a little better since I didn't know if it was just me being silly for not being able to get it working.

I still like this approach more than the FW software assistant so I'll play around with it myself as well and if I come up with something I'll submit a PR.
